monitoring/osd-configuring-the-monitoring-stack.adoc

```diff
@@ -0,0 +1,115 @@
+
+[IMPORTANT]
+====
+See link:https://docs.openshift.com/container-platform/4.6/scalability_and_performance/optimizing-storage.html#recommended-configurable-storage-technology_persistent-storage[Recommended configurable storage technology].
+====
+
+[id="persistent-storage-prerequisites"]
+=== Persistent storage prerequisites
+
+* Dedicate sufficient local persistent storage to ensure that the disk does not become full. How much storage you need depends on the number of pods. For information on system requirements for persistent storage, see xref:../scalability_and_performance/scaling-cluster-monitoring-operator.adoc#prometheus-database-storage-requirements[Prometheus database storage requirements].
+
+* Make sure you have a persistent volume (PV) ready to be claimed by the persistent volume claim (PVC), one PV for each replica. Because Prometheus has two replicas and Alertmanager has three replicas, you need five PVs to support the entire monitoring stack. The PVs should be available from the Local Storage Operator. This does not apply if you enable dynamically provisioned storage.
+
+* Use the block type of storage.
+
+* link:https://docs.openshift.com/container-platform/4.6/storage/persistent_storage/persistent-storage-local.html[Configure local persistent storage.]
+
+include::modules/osd-monitoring-configuring-a-local-persistent-volume-claim.adoc[leveloffset=+2]
+include::modules/osd-monitoring-modifying-retention-time-for-prometheus-metrics-data.adoc[leveloffset=+2]
+
+.Additional resources
+
+* See xref:../monitoring/osd-configuring-the-monitoring-stack.adoc#preparing-to-configure-the-monitoring-stack[Preparing to configure the monitoring stack] for steps to create monitoring config maps
+* xref:../monitoring/osd-accessing-monitoring-for-user-defined-projects.adoc[Accessing monitoring for user-defined projects]
+* link:https://docs.openshift.com/container-platform/4.6/storage/understanding-persistent-storage.html[Understanding persistent storage]
+* link:https://docs.openshift.com/container-platform/4.6/scalability_and_performance/optimizing-storage.html[Optimizing storage]
+
+// Managing scrape sample limits for user-defined projects
+include::modules/osd-monitoring-limiting-scrape-samples-in-user-defined-projects.adoc[leveloffset=+1]
+include::modules/osd-monitoring-setting-a-scrape-sample-limit-for-user-defined-projects.adoc[leveloffset=+2]
+include::modules/osd-monitoring-creating-scrape-sample-alerts.adoc[leveloffset=+2]
+
+.Additional resources
+
+* xref:../monitoring/osd-accessing-monitoring-for-user-defined-projects.adoc[Accessing monitoring for user-defined projects]
+* link:https://docs.openshift.com/container-platform/4.6/monitoring/troubleshooting-monitoring-issues.html#determining-why-prometheus-is-consuming-disk-space_troubleshooting-monitoring-issues[Determining why Prometheus is consuming a lot of disk space] for steps to query which metrics have the highest number of scrape samples
```

Comment:
This is a link that directs the reader to the OpenShift Container Platform documentation. Is this relevant to ROSA, or should this link be removed? The reader might not realise that they are being directed to the docs for a different platform type.

Reply:
Kept the link. Removed the Xref.